docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -12,67 +12,170 @@ isTocNested: true
+- Enter the file paths for each of the certificates on the nodes. These are the paths from the previous step.
+- In the **Certificate Name** field, enter a meaningful name for your certificate.
+- Use the **Root Certificate Expiration Date** field to specify the expiration date of the certificate. To find this information, execute the `openssl x509 -in <root crt file path> -text -noout` command and note the Validity Not After date.
+- Click **Add** to make the certificate available.
+- Go to **Universes > Create Universe** to open the **Create Universe** dialog.
+- Configure the universe.
+- Based on your requirements, select **Enable Node-to-Node TLS** and **Enable Client-to-Node TLS**.
+- Select an existing certificate from the **Root Certificate** list and then select the certificate that you have uploaded.
+- Create the universe.
+
+### How to Rotate Custom CA-Signed Certificates
+
+You can rotate custom CA-signed certificates for universes configured with these type of certificates.
+
+The following procedure describes how to rotate the existing custom certificates and replace them with new database node certificates issued by the same custom CA that issued the original certificates. Note that in the current release, Yugabyte Platform does not support changing the root CA certificate.
+
+**Step 1**: Follow Step 1 of [How to Use Custom CA-Signed Certificates to Enable TLS](#how-to-use-custom-ca-signed-certificates-to-enable-tls) to obtain a new set of certificates for each of the nodes.
+
+**Step 2**: Follow Step 2 of  [How to Use Custom CA-Signed Certificates to Enable TLS](#how-to-use-custom-ca-signed-certificates-to-enable-tls) to copy the certificates to the respective nodes. 
+
+**Step 3**: Follow Step 3 of [How to Use Custom CA-Signed Certificates to Enable TLS](#how-to-use-custom-ca-signed-certificates-to-enable-tls) to create a new CA-signed certificate in Yugabyte Platform.
+
+**Step 4**: Edit the universe to use the new certificates, as follows:
+
+- Navigate to the universe for which you are rotating the keys.
+- Select **Actions > Edit Security**, as shown in the following illustration. 
+
+![edit-security](/images/yp/encryption-in-transit/edit-security.png)   
+
+- Select **Encryption in-Transit**, as shown in the following illustration. 
+
+![encryption-in-transit](/images/yp/encryption-in-transit/encryption-in-transit.png)  
+
+- Complete the Edit TLS Configuration dialog shown in the following illustration:
+  - Select the new certificate which you created in Step 3.
+  - Modifying the certificates requires restarting Master and T-Server processes. To avoid downtime, select **Rolling Upgrade** and specify the delay between node upgrades.
+  - Click **Upgrade**. Typically, this process takes time as it needs to wait for the specified delay interval after each node is upgraded.
+
+![edit-tls](/images/yp/encryption-in-transit/edit-tls.png)    
+
+
+
+### How to Expand the Universe
```

Comment:
@lizayugabyte Please copy-paste this section "How to Expand the Universe" here and mention that this step is optional for universe configured with custom CA-signed certs onlu.

https://docs.yugabyte.com/latest/yugabyte-platform/manage-deployments/edit-universe/

Reply:
@chirag-yb  This can create doc maintenance issues: if the procedure changes, we'd have to remember to edit the content in two different files located in different folders. I would suggest adding a link from edit-universe.md to the "How to Expand the Universe" section of enable-encryption-in-transit.md. What do you think?

Reply:
@chirag-yb  I implemented your first two comments and for the third one, I added a link. Could you, please, let me know if that's okay so I can close the ticket?

Reply:
@chirag-yb  Based on our conversation, I added the following sentence with link to edit-universe.md, just below the image: _For information on how to expand a universe created with an on-premise cloud provider and secured with third-party certificates obtained from external CAs, see How to Expand the Universe._